lnwallet/channel.go

```diff
@@ -3092,6 +3279,89 @@ func (lc *LightningChannel) createCommitDiff(
 	}, nil
 }
 
+// getDanglingRemoteUpdates returns all remote log updates that we haven't
+// signed for yet ourselves.
+func (lc *LightningChannel) getDanglingRemoteUpdates() []channeldb.LogUpdate {
+	// First, we need to convert the funding outpoint into the ID that's
+	// used on the wire to identify this channel.
+	chanID := lnwire.NewChanIDFromOutPoint(&lc.channelState.FundingOutpoint)
+
+	// Fetch the last remote update that we have added to our remote
+	// commitment chain.
+	chainTail := lc.remoteCommitChain.tail()
```

Comment:
should be tip?

Reply:
good catch. we should at some point replace the list by a single pointer, now that the revocation window is always max 1.